gfx/drivers_context/drm_ctx.c

```diff
@@ -215,9 +215,10 @@ static bool gfx_ctx_drm_queue_flip(void)
    fb                = (struct drm_fb*)drm_fb_get_from_bo(g_next_bo);
 
    if (drmModePageFlip(g_drm_fd, g_crtc_id, fb->fb_id,
-         DRM_MODE_PAGE_FLIP_EVENT, &waiting_for_flip) == 0)
+                       DRM_MODE_PAGE_FLIP_EVENT | DRM_MODE_PAGE_FLIP_ASYNC,
```

Comment:
This is the only change I see. Can you explain what this does, and what it accomplishes? I haven't seen this flag before.